frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -67,6 +67,12 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		.any(|c| matches!(c.composite_keyword, CompositeKeyword::HoldReason(_)))
 		.then_some(quote::quote!(HoldReason,));
 
+	let task_part = def
+		.composites
```

Comment:
This looks wrong. `Task` is not declared using `#[pallet::composite_enum]`, so it shouldn't be accessing `composites` here. Instead, a new field called `task` should exist on the `Def`.

Reply:
ahhh that's probably it.

Reply:
although this isn't the current issue -- this would affect things during expand but I'm having a compile fail during parse